betterdgg/modules/mentions.js

```diff
@@ -7,9 +7,12 @@
                 gui.lines.on('mousedown', 'div.user-msg a.user', function() {
                     if (bdgg.settings.get('bdgg_highlight_selected_mentions')) {
                         var username = $(this).text();
-                        gui.lines.find(':contains(' + username + ')').addClass('focused');
+                        var users = [$(this).parents().parents().find(".chat-user:contains(" + username + ")")];
+                        for (var i = 0; i < $(users)[0].length - 1; i++) {
+                            window.destiny.chat.gui.stylesheet.insertRule('.user-msg[data-username="' + ($($(users)[0].parent().parent()[i]).find(".user").text().toLowerCase()) + '"]{opacity:1 !important;}', window.destiny.chat.gui.stylesheet.cssRules.length);
+                            window.destiny.chat.gui.focusedUsers.push(($($(users)[0].parent().parent()[i]).find(".user").text().toLowerCase()));
```

Comment:
I think I understand now that

``` js
($($(users)[0].parent().parent()[i]).find(".user").text().toLowerCase())
```

gives us the `i`th user to mention `username`. Could we make this its own variable in the loop, so that the lines aren't so long and also to help with readability?